packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -336,6 +344,30 @@ export interface TreeAlpha {
 	 * Otherwise, this returns the key of the field that it is under (a `string`).
 	 */
 	key2(node: TreeNode): string | number | undefined;
+
+	/**
+	 * Gets the child of the given node with the given key if a child exists under that key.
```

Comment:
I think both of these new APIs should probably mention TreeAlpha.key2 and Tree.parent (and those should probably link at least TreAlpha.child)